src/runtime_src/core/common/smi.cpp

```diff
@@ -0,0 +1,263 @@
+// SPDX-License-Identifier: Apache-2.0
+// Copyright (C) 2025 Advanced Micro Devices, Inc. All rights reserved.
+#include "smi.h"
+#include <boost/property_tree/ptree.hpp>
+#include <boost/property_tree/json_parser.hpp>
+#include <vector>
+#include <tuple>
+#include <string>
+#include <boost/property_tree/ptree.hpp>
+#include <boost/property_tree/json_parser.hpp>
```

Comment:
Sort in groups.  Keep smi.h first, then boost includes and finally system headers